test/e2e/config/fixture.yaml

```
@@ -32,3 +36,6 @@ intervals:
   default/wait-deployment: ["5m", "1s"]
   default/wait-namespace-deleted: ["20s", "1s"]
   ironic/wait-deployment: ["10m", "2s"]
+  default/wait-provisioning: ["20s", "1s"]
+  default/wait-provisioned: ["5m", "5s"]
```

This seems excessive. The whole test took ~25 seconds with the fixture provider so I think we can lower both the `5m` deadline and the `5s` interval.
```suggestion
  default/wait-provisioned: ["1m", "10ms"]
```